src/main/java/ch/njol/skript/conditions/CondScriptLoaded.java

```diff
@@ -69,13 +75,13 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 	@Override
 	public boolean check(Event e) {
 		if (scripts == null) {
-			if (currentScript == null)
-				return isNegated();
+			assert currentScript != null; // If scripts weren't provided then currentScript can't be null
```

Comment:
Remember this won't be available outside a testing environment - you might want to throw a real assertion error if this is actually an important check.